test/models/commits_test.rb

```diff
@@ -65,11 +65,15 @@ class CommitsTest < ActiveSupport::TestCase
     end
 
     test "updating state to success triggers new deploy when stack has continuous deployment" do
+      Hook.destroy_all
```

Comment:
It's not anymore. It was to prevent hooks from firing and generating network calls. But since then I changed the test to only run `ContinuousDeliveryJob`, so I'll :fire: that too.